src/store/modules/currentFarmer/types.ts

```diff
@@ -0,0 +1,14 @@
+/*----------------------- State -----------------------*/
+/** currentFarmer State*/
+export interface CurrentFarmerState{
+  currentFarmerUUID: string;
+}
+
+/*----------------------- Actions -----------------------*/
+/** currentFarmer Action Types */
+export type Action = {
+  type: 'UPDATE_CURRENT_FARMER',
+  currentFarmerUUID: string,
```

Comment:
I think there should be an action to clear the active farmer to avoid future bugs. Either set to null or `''` when no active farmer

Reply:
Bea and I discussed this. Realistically you are just updating it to ' '. I had a delete current farmer action but it sounded really scary

Reply:
Clear current farmer would be more appropriate because we aren't actually deleting

Reply:
You're going to hate me but I think I found a way to avoid doing this

Reply:
How do you want me to tackle this one. Shall I add a clear action or did you do it another way?

Reply:
For now leave your stuff in, but there is a way to add routing parameters for us to use